pkg/minikube/config/profile.go

```diff
@@ -301,6 +301,10 @@ func profileDirs(miniHome ...string) (dirs []string, err error) {
 	}
 	pRootDir := filepath.Join(miniPath, "profiles")
 	items, err := os.ReadDir(pRootDir)
+	if err == os.IsNotExist(err) {
+		return dirs, &ErrNotExist{fmt.Sprintf("profiles dir %s does not exist", pRootDir)}
+	}
```

Comment:
I think the best way to handle this is to return here empty list of directories and no error since this is an expected condition before any profile was created, or if a user deleted the profiles directory.

However if we report an error, we should do the check correctly - it seems that you are adding here a wrong check:

    if err == os.IsNotExist(err)

And you fix it in the next commit? Did you forget to squash the next commit into this?